plugin-server/src/worker/ingestion/process-event.ts

```diff
@@ -593,7 +599,13 @@ export class EventsProcessor {
             const useExternalSchemas = this.clickhouseExternalSchemasEnabled(teamId)
             const message = useExternalSchemas
```

Comment:
let's add a comment about how proto ingestion is deprecated and won't support new additions to the schema